charts/fleet/templates/deployment.yaml

```diff
@@ -33,7 +33,7 @@ spec:
       - name: fleet
         command: [/usr/bin/fleet]
         args: ["serve"]
-        image: fleetdm/fleet:{{ .Values.imageTag }}
+        image: "{{ .Values.image }}:{{ .Values.imageTag }}"
```

Comment:
Hi there and thanks for the contribution! Normally this would be called the image.repository and what we have as 'imageTag' would be combined together into the 'image'. Just so people aren't confused if this is also setting the tag would you mind renaming this value key to imageRepository?

Reply:
Done. Tested locally in my k8s environment too.